frontend/leads-list/table/desktop-leads-table.component.tsx

```diff
@@ -324,7 +324,7 @@ const css = `
   }
 
   & .leads-table.advanced-search th {
-    top: 47.5;
+    top: 45rem;
```

Comment:
In my comment on Github, I think I made a mistake. I did not mean to change from `47.5` to `45` - just meant to add the `rem`. Were you able to confirm whether 47.5 or 45 is the correct value for scroll when advanced search is expanded?

Reply:
I confirmed this myself today. Looks good.